charts/gateway-addons-helm/values.yaml

```diff
@@ -1,9 +1,3 @@
-tags:
```

Comment:
remove tags since we already explicitly set conditions for these dependencies, and conditions can override tags according to the [helm doc](https://helm.sh/docs/topics/charts/#tags-and-condition-fields-in-dependencies)